src/actions/index.js

```diff
@@ -7,3 +9,46 @@ export const VisibilityFilters = {
   SHOW_ALL: 'SHOW_ALL',
   SHOW_SAVED: 'SHOW_SAVED',
 };
+
+// action creators
+
+export function itemsHasErrored(bool) {
```

Comment:
you can take a look at redux-ducks and for FSA standart for the action and later refactor the code to them

Reply:
I should follow one of these patterns to write the actions.

Reply:
I'm sure that it worth to follow both patterns )

Reply:
If I have more than one action in the file. In this app I have three actions for three different property of app state, there are: `itemsHasErrored`, `itemsIsLoading`, `itemsFetchDataSuccess`. And all these actions work with one thing, with API. And all actions located in one file, and all reducers is located in one file. (Other reducers located in other files).

If I'll follow the ducks pattern I should to located all reducers in different files. And all actions shold be located in different files. And I think that it is too much.

I suggest this:
![image](https://user-images.githubusercontent.com/38797690/55948751-da8c8e00-5c59-11e9-94e1-15f68edd8452.png)

Reply:
@AlexandrTichon Why the api reducer cant be duck? We can call each other and discuss the question by voice )